src/views/partials/mobile-nav.njk

```diff
@@ -0,0 +1,23 @@
+<nav class="hidden bg-grey-5" id="nav-mobile">
+    <ul>
+        {% for section in sections %}
+            <li>
+                {% if section.title %}
+                    <button class="text-base border-b border-grey-4 hover:bg-grey-4 text-left w-full text-grey-1 px-4 p-3 pr-2 flex items-center justify-between nav-mobile-btn" data-index="{{loop.index0}}" >{{section.title}}
+                        <svg class="transform -rotate-90" width="20" height="20" viewBox="0 0 20 20" fill="currentColor" xmlns="http://www.w3.org/2000/svg" aria-hidden="true"><path d="M5 6H15L10 15L5 6Z"/></svg>
+                    </button>
+                {% endif %}
+                <ul class="md:mb-4 {% if section.title %} hidden {% endif %}" id="nav-mobile-section-{{loop.index0}}">
+                    {% for post in section.pages %}
+                        {% set isActive = page.url == post.url %}
+                        <li class="border-b border-grey-4">
+                            <a href="{{post.url | url }}" class="block px-4 py-3 no-underline hover:bg-grey-4 border-ocean-blue border-opacity-0 border-l-4 border-inset  {% if isActive %} border-opacity-100  {% endif %} " {% if isActive %} aria-current="page" {% endif %}>
```

Comment:
Nothing major just an observation some files have spaces on the inside of the variable lookups `{{ something }}` (rather than `{{something}}` think typical njk is to have it the spaces

Reply:
I think there's more instances without spaces than with them. I'm surprised Prettier isn't picking this up (I have "format on save" turned on).